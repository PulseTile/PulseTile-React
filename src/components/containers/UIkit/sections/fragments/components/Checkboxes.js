import React, { Component } from 'react';

/**
 * This component returns content of Checkboxes section
 *
 * @return {XML}
 * @constructor
 */
export default class Checkboxes extends Component {

  state = {
    isCheckboxTest1: false,
    isCheckboxTest2: false,
    isCheckboxInlineTest1: false,
    isCheckboxInlineTest2: false,
    isCheckboxInlineTest3: false,
  };

  checkPosition = current => {
    this.setState({
      [current] : !this.state[current]
    });
  };

  render() {
    const { isCheckboxTest1, isCheckboxTest2, isCheckboxInlineTest1, isCheckboxInlineTest2, isCheckboxInlineTest3 } = this.state;
    return (
      <div id="checkboxes" className="ui-section">
        <strong className="ui-title">Checkboxes</strong>
        <div className="form-group-wrapper">
          <div className="form-group">
            <div className="input-holder">
              <div className="input-holder">
                <div className="wrap-fcustominp">
                  <div className="fcustominp" onClick={() => this.checkPosition('isCheckboxTest1')}>
                    <input type="checkbox" id="checkboxTest1" checked={isCheckboxTest1} />
                    <label for="checkboxTest1"></label>
                  </div>
                  <label className="fcustominp-label">Checkbox Test 1</label>
                </div>
               </div>
             </div>
           </div>
           <div className="form-group">
             <div className="input-holder">
               <div className="input-holder">
                 <div className="wrap-fcustominp">
                   <div className="fcustominp" onClick={() => this.checkPosition('isCheckboxTest2')}>
                     <input type="checkbox" id="checkboxTest2" checked={isCheckboxTest2}  />
                     <label for="checkboxTest2"></label>
                   </div>
                   <label className="fcustominp-label">Checkbox Test 2</label>
                 </div>
               </div>
             </div>
           </div>
           <div className="form-group">
             <div className="input-holder">
               <div className="input-holder">
                 <div className="wrap-fcustominp">
                   <div className="fcustominp-state disabled">
                     <div className="fcustominp">
                       <input type="checkbox" id="checkboxTest6" disabled />
                       <label for="radioTest6"></label>
                     </div>
                     <label for="radioTest6" className="fcustominp-label">Checkbox Test 3</label>
                   </div>
                 </div>
               </div>
             </div>
            </div>
            <div className="form-group">
              <div className="input-holder">
                <div className="input-holder">
                  <div className="wrap-fcustominps-inline">
                    <div className="wrap-fcustominp">
                      <div className="fcustominp" onClick={() => this.checkPosition('isCheckboxInlineTest1')}>
                        <input type="checkbox" id="checkboxTest3" checked={isCheckboxInlineTest1} />
                        <label for="checkboxTest3"></label>
                      </div>
                      <label className="fcustominp-label">Checkbox Inline Test 1</label>
                    </div>
                    <div className="wrap-fcustominp">
                      <div className="fcustominp" onClick={() => this.checkPosition('isCheckboxInlineTest2')}>
                        <input type="checkbox" id="checkboxTest4" checked={isCheckboxInlineTest2} />
                        <label for="checkboxTest4"></label>
                      </div>
                      <label className="fcustominp-label">Checkbox Inline Test 2</label>
                    </div>
                    <div className="wrap-fcustominp">
                      <div className="fcustominp" onClick={() => this.checkPosition('isCheckboxInlineTest3')}>
                        <input type="checkbox" id="checkboxTest5" checked={isCheckboxInlineTest3} />
                        <label for="checkboxTest5"></label>
                      </div>
                      <label className="fcustominp-label">Checkbox Inline Test 3</label>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      );
  }
}