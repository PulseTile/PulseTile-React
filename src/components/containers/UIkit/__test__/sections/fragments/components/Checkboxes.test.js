import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-15';
import Checkboxes from '../../../../sections/fragments/components/Checkboxes';

Enzyme.configure({ adapter: new Adapter() });

const CHECKBOXES_NUMBER = 6;

describe('Component <Checkboxes />', () => {
  it('should renders Checkboxes with props correctly', () => {
    const component = shallow(<Checkboxes />);
    expect(component).toMatchSnapshot();
    for (let i = 0; i < CHECKBOXES_NUMBER; i++) {
      component.find('.fcustominp').at(i).simulate('click');
      expect(component).toMatchSnapshot();
      component.find('.fcustominp').at(i).simulate('click');
    }
    expect(component).toMatchSnapshot();
  });
});
