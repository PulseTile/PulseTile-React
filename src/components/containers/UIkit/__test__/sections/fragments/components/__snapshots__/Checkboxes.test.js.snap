// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component <Checkboxes /> should renders Checkboxes with props correctly 1`] = `
<div
  className="ui-section"
  id="checkboxes"
>
  <strong
    className="ui-title"
  >
    Checkboxes
  </strong>
  <div
    className="form-group-wrapper"
  >
    <div
      className="form-group"
    >
      <div
        className="input-holder"
      >
        <div
          className="input-holder"
        >
          <div
            className="wrap-fcustominp"
          >
            <div
              className="fcustominp"
              onClick={[Function]}
            >
              <input
                checked={false}
                id="checkboxTest1"
                type="checkbox"
              />
              <label
                for="checkboxTest1"
              />
            </div>
            <label
              className="fcustominp-label"
            >
              Checkbox Test 1
            </label>
          </div>
        </div>
      </div>
    </div>
    <div
      className="form-group"
    >
      <div
        className="input-holder"
      >
        <div
          className="input-holder"
        >
          <div
            className="wrap-fcustominp"
          >
            <div
              className="fcustominp"
              onClick={[Function]}
            >
              <input
                checked={false}
                id="checkboxTest2"
                type="checkbox"
              />
              <label
                for="checkboxTest2"
              />
            </div>
            <label
              className="fcustominp-label"
            >
              Checkbox Test 2
            </label>
          </div>
        </div>
      </div>
    </div>
    <div
      className="form-group"
    >
      <div
        className="input-holder"
      >
        <div
          className="input-holder"
        >
          <div
            className="wrap-fcustominp"
          >
            <div
              className="fcustominp-state disabled"
            >
              <div
                className="fcustominp"
              >
                <input
                  disabled={true}
                  id="checkboxTest6"
                  type="checkbox"
                />
                <label
                  for="radioTest6"
                />
              </div>
              <label
                className="fcustominp-label"
                for="radioTest6"
              >
                Checkbox Test 3
              </label>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      className="form-group"
    >
      <div
        className="input-holder"
      >
        <div
          className="input-holder"
        >
          <div
            className="wrap-fcustominps-inline"
          >
            <div
              className="wrap-fcustominp"
            >
              <div
                className="fcustominp"
                onClick={[Function]}
              >
                <input
                  checked={false}
                  id="checkboxTest3"
                  type="checkbox"
                />
                <label
                  for="checkboxTest3"
                />
              </div>
              <label
                className="fcustominp-label"
              >
                Checkbox Inline Test 1
              </label>
            </div>
            <div
              className="wrap-fcustominp"
            >
              <div
                className="fcustominp"
                onClick={[Function]}
              >
                <input
                  checked={false}
                  id="checkboxTest4"
                  type="checkbox"
                />
                <label
                  for="checkboxTest4"
                />
              </div>
              <label
                className="fcustominp-label"
              >
                Checkbox Inline Test 2
              </label>
            </div>
            <div
              className="wrap-fcustominp"
            >
              <div
                className="fcustominp"
                onClick={[Function]}
              >
                <input
                  checked={false}
                  id="checkboxTest5"
                  type="checkbox"
                />
                <label
                  for="checkboxTest5"
                />
              </div>
              <label
                className="fcustominp-label"
              >
                Checkbox Inline Test 3
              </label>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Component <Checkboxes /> should renders Checkboxes with props correctly 2`] = `
<div
  className="ui-section"
  id="checkboxes"
>
  <strong
    className="ui-title"
  >
    Checkboxes
  </strong>
  <div
    className="form-group-wrapper"
  >
    <div
      className="form-group"
    >
      <div
        className="input-holder"
      >
        <div
          className="input-holder"
        >
          <div
            className="wrap-fcustominp"
          >
            <div
              className="fcustominp"
              onClick={[Function]}
            >
              <input
                checked={true}
                id="checkboxTest1"
                type="checkbox"
              />
              <label
                for="checkboxTest1"
              />
            </div>
            <label
              className="fcustominp-label"
            >
              Checkbox Test 1
            </label>
          </div>
        </div>
      </div>
    </div>
    <div
      className="form-group"
    >
      <div
        className="input-holder"
      >
        <div
          className="input-holder"
        >
          <div
            className="wrap-fcustominp"
          >
            <div
              className="fcustominp"
              onClick={[Function]}
            >
              <input
                checked={false}
                id="checkboxTest2"
                type="checkbox"
              />
              <label
                for="checkboxTest2"
              />
            </div>
            <label
              className="fcustominp-label"
            >
              Checkbox Test 2
            </label>
          </div>
        </div>
      </div>
    </div>
    <div
      className="form-group"
    >
      <div
        className="input-holder"
      >
        <div
          className="input-holder"
        >
          <div
            className="wrap-fcustominp"
          >
            <div
              className="fcustominp-state disabled"
            >
              <div
                className="fcustominp"
              >
                <input
                  disabled={true}
                  id="checkboxTest6"
                  type="checkbox"
                />
                <label
                  for="radioTest6"
                />
              </div>
              <label
                className="fcustominp-label"
                for="radioTest6"
              >
                Checkbox Test 3
              </label>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      className="form-group"
    >
      <div
        className="input-holder"
      >
        <div
          className="input-holder"
        >
          <div
            className="wrap-fcustominps-inline"
          >
            <div
              className="wrap-fcustominp"
            >
              <div
                className="fcustominp"
                onClick={[Function]}
              >
                <input
                  checked={false}
                  id="checkboxTest3"
                  type="checkbox"
                />
                <label
                  for="checkboxTest3"
                />
              </div>
              <label
                className="fcustominp-label"
              >
                Checkbox Inline Test 1
              </label>
            </div>
            <div
              className="wrap-fcustominp"
            >
              <div
                className="fcustominp"
                onClick={[Function]}
              >
                <input
                  checked={false}
                  id="checkboxTest4"
                  type="checkbox"
                />
                <label
                  for="checkboxTest4"
                />
              </div>
              <label
                className="fcustominp-label"
              >
                Checkbox Inline Test 2
              </label>
            </div>
            <div
              className="wrap-fcustominp"
            >
              <div
                className="fcustominp"
                onClick={[Function]}
              >
                <input
                  checked={false}
                  id="checkboxTest5"
                  type="checkbox"
                />
                <label
                  for="checkboxTest5"
                />
              </div>
              <label
                className="fcustominp-label"
              >
                Checkbox Inline Test 3
              </label>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Component <Checkboxes /> should renders Checkboxes with props correctly 3`] = `
<div
  className="ui-section"
  id="checkboxes"
>
  <strong
    className="ui-title"
  >
    Checkboxes
  </strong>
  <div
    className="form-group-wrapper"
  >
    <div
      className="form-group"
    >
      <div
        className="input-holder"
      >
        <div
          className="input-holder"
        >
          <div
            className="wrap-fcustominp"
          >
            <div
              className="fcustominp"
              onClick={[Function]}
            >
              <input
                checked={false}
                id="checkboxTest1"
                type="checkbox"
              />
              <label
                for="checkboxTest1"
              />
            </div>
            <label
              className="fcustominp-label"
            >
              Checkbox Test 1
            </label>
          </div>
        </div>
      </div>
    </div>
    <div
      className="form-group"
    >
      <div
        className="input-holder"
      >
        <div
          className="input-holder"
        >
          <div
            className="wrap-fcustominp"
          >
            <div
              className="fcustominp"
              onClick={[Function]}
            >
              <input
                checked={true}
                id="checkboxTest2"
                type="checkbox"
              />
              <label
                for="checkboxTest2"
              />
            </div>
            <label
              className="fcustominp-label"
            >
              Checkbox Test 2
            </label>
          </div>
        </div>
      </div>
    </div>
    <div
      className="form-group"
    >
      <div
        className="input-holder"
      >
        <div
          className="input-holder"
        >
          <div
            className="wrap-fcustominp"
          >
            <div
              className="fcustominp-state disabled"
            >
              <div
                className="fcustominp"
              >
                <input
                  disabled={true}
                  id="checkboxTest6"
                  type="checkbox"
                />
                <label
                  for="radioTest6"
                />
              </div>
              <label
                className="fcustominp-label"
                for="radioTest6"
              >
                Checkbox Test 3
              </label>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      className="form-group"
    >
      <div
        className="input-holder"
      >
        <div
          className="input-holder"
        >
          <div
            className="wrap-fcustominps-inline"
          >
            <div
              className="wrap-fcustominp"
            >
              <div
                className="fcustominp"
                onClick={[Function]}
              >
                <input
                  checked={false}
                  id="checkboxTest3"
                  type="checkbox"
                />
                <label
                  for="checkboxTest3"
                />
              </div>
              <label
                className="fcustominp-label"
              >
                Checkbox Inline Test 1
              </label>
            </div>
            <div
              className="wrap-fcustominp"
            >
              <div
                className="fcustominp"
                onClick={[Function]}
              >
                <input
                  checked={false}
                  id="checkboxTest4"
                  type="checkbox"
                />
                <label
                  for="checkboxTest4"
                />
              </div>
              <label
                className="fcustominp-label"
              >
                Checkbox Inline Test 2
              </label>
            </div>
            <div
              className="wrap-fcustominp"
            >
              <div
                className="fcustominp"
                onClick={[Function]}
              >
                <input
                  checked={false}
                  id="checkboxTest5"
                  type="checkbox"
                />
                <label
                  for="checkboxTest5"
                />
              </div>
              <label
                className="fcustominp-label"
              >
                Checkbox Inline Test 3
              </label>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Component <Checkboxes /> should renders Checkboxes with props correctly 4`] = `
<div
  className="ui-section"
  id="checkboxes"
>
  <strong
    className="ui-title"
  >
    Checkboxes
  </strong>
  <div
    className="form-group-wrapper"
  >
    <div
      className="form-group"
    >
      <div
        className="input-holder"
      >
        <div
          className="input-holder"
        >
          <div
            className="wrap-fcustominp"
          >
            <div
              className="fcustominp"
              onClick={[Function]}
            >
              <input
                checked={false}
                id="checkboxTest1"
                type="checkbox"
              />
              <label
                for="checkboxTest1"
              />
            </div>
            <label
              className="fcustominp-label"
            >
              Checkbox Test 1
            </label>
          </div>
        </div>
      </div>
    </div>
    <div
      className="form-group"
    >
      <div
        className="input-holder"
      >
        <div
          className="input-holder"
        >
          <div
            className="wrap-fcustominp"
          >
            <div
              className="fcustominp"
              onClick={[Function]}
            >
              <input
                checked={false}
                id="checkboxTest2"
                type="checkbox"
              />
              <label
                for="checkboxTest2"
              />
            </div>
            <label
              className="fcustominp-label"
            >
              Checkbox Test 2
            </label>
          </div>
        </div>
      </div>
    </div>
    <div
      className="form-group"
    >
      <div
        className="input-holder"
      >
        <div
          className="input-holder"
        >
          <div
            className="wrap-fcustominp"
          >
            <div
              className="fcustominp-state disabled"
            >
              <div
                className="fcustominp"
              >
                <input
                  disabled={true}
                  id="checkboxTest6"
                  type="checkbox"
                />
                <label
                  for="radioTest6"
                />
              </div>
              <label
                className="fcustominp-label"
                for="radioTest6"
              >
                Checkbox Test 3
              </label>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      className="form-group"
    >
      <div
        className="input-holder"
      >
        <div
          className="input-holder"
        >
          <div
            className="wrap-fcustominps-inline"
          >
            <div
              className="wrap-fcustominp"
            >
              <div
                className="fcustominp"
                onClick={[Function]}
              >
                <input
                  checked={false}
                  id="checkboxTest3"
                  type="checkbox"
                />
                <label
                  for="checkboxTest3"
                />
              </div>
              <label
                className="fcustominp-label"
              >
                Checkbox Inline Test 1
              </label>
            </div>
            <div
              className="wrap-fcustominp"
            >
              <div
                className="fcustominp"
                onClick={[Function]}
              >
                <input
                  checked={false}
                  id="checkboxTest4"
                  type="checkbox"
                />
                <label
                  for="checkboxTest4"
                />
              </div>
              <label
                className="fcustominp-label"
              >
                Checkbox Inline Test 2
              </label>
            </div>
            <div
              className="wrap-fcustominp"
            >
              <div
                className="fcustominp"
                onClick={[Function]}
              >
                <input
                  checked={false}
                  id="checkboxTest5"
                  type="checkbox"
                />
                <label
                  for="checkboxTest5"
                />
              </div>
              <label
                className="fcustominp-label"
              >
                Checkbox Inline Test 3
              </label>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Component <Checkboxes /> should renders Checkboxes with props correctly 5`] = `
<div
  className="ui-section"
  id="checkboxes"
>
  <strong
    className="ui-title"
  >
    Checkboxes
  </strong>
  <div
    className="form-group-wrapper"
  >
    <div
      className="form-group"
    >
      <div
        className="input-holder"
      >
        <div
          className="input-holder"
        >
          <div
            className="wrap-fcustominp"
          >
            <div
              className="fcustominp"
              onClick={[Function]}
            >
              <input
                checked={false}
                id="checkboxTest1"
                type="checkbox"
              />
              <label
                for="checkboxTest1"
              />
            </div>
            <label
              className="fcustominp-label"
            >
              Checkbox Test 1
            </label>
          </div>
        </div>
      </div>
    </div>
    <div
      className="form-group"
    >
      <div
        className="input-holder"
      >
        <div
          className="input-holder"
        >
          <div
            className="wrap-fcustominp"
          >
            <div
              className="fcustominp"
              onClick={[Function]}
            >
              <input
                checked={false}
                id="checkboxTest2"
                type="checkbox"
              />
              <label
                for="checkboxTest2"
              />
            </div>
            <label
              className="fcustominp-label"
            >
              Checkbox Test 2
            </label>
          </div>
        </div>
      </div>
    </div>
    <div
      className="form-group"
    >
      <div
        className="input-holder"
      >
        <div
          className="input-holder"
        >
          <div
            className="wrap-fcustominp"
          >
            <div
              className="fcustominp-state disabled"
            >
              <div
                className="fcustominp"
              >
                <input
                  disabled={true}
                  id="checkboxTest6"
                  type="checkbox"
                />
                <label
                  for="radioTest6"
                />
              </div>
              <label
                className="fcustominp-label"
                for="radioTest6"
              >
                Checkbox Test 3
              </label>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      className="form-group"
    >
      <div
        className="input-holder"
      >
        <div
          className="input-holder"
        >
          <div
            className="wrap-fcustominps-inline"
          >
            <div
              className="wrap-fcustominp"
            >
              <div
                className="fcustominp"
                onClick={[Function]}
              >
                <input
                  checked={true}
                  id="checkboxTest3"
                  type="checkbox"
                />
                <label
                  for="checkboxTest3"
                />
              </div>
              <label
                className="fcustominp-label"
              >
                Checkbox Inline Test 1
              </label>
            </div>
            <div
              className="wrap-fcustominp"
            >
              <div
                className="fcustominp"
                onClick={[Function]}
              >
                <input
                  checked={false}
                  id="checkboxTest4"
                  type="checkbox"
                />
                <label
                  for="checkboxTest4"
                />
              </div>
              <label
                className="fcustominp-label"
              >
                Checkbox Inline Test 2
              </label>
            </div>
            <div
              className="wrap-fcustominp"
            >
              <div
                className="fcustominp"
                onClick={[Function]}
              >
                <input
                  checked={false}
                  id="checkboxTest5"
                  type="checkbox"
                />
                <label
                  for="checkboxTest5"
                />
              </div>
              <label
                className="fcustominp-label"
              >
                Checkbox Inline Test 3
              </label>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Component <Checkboxes /> should renders Checkboxes with props correctly 6`] = `
<div
  className="ui-section"
  id="checkboxes"
>
  <strong
    className="ui-title"
  >
    Checkboxes
  </strong>
  <div
    className="form-group-wrapper"
  >
    <div
      className="form-group"
    >
      <div
        className="input-holder"
      >
        <div
          className="input-holder"
        >
          <div
            className="wrap-fcustominp"
          >
            <div
              className="fcustominp"
              onClick={[Function]}
            >
              <input
                checked={false}
                id="checkboxTest1"
                type="checkbox"
              />
              <label
                for="checkboxTest1"
              />
            </div>
            <label
              className="fcustominp-label"
            >
              Checkbox Test 1
            </label>
          </div>
        </div>
      </div>
    </div>
    <div
      className="form-group"
    >
      <div
        className="input-holder"
      >
        <div
          className="input-holder"
        >
          <div
            className="wrap-fcustominp"
          >
            <div
              className="fcustominp"
              onClick={[Function]}
            >
              <input
                checked={false}
                id="checkboxTest2"
                type="checkbox"
              />
              <label
                for="checkboxTest2"
              />
            </div>
            <label
              className="fcustominp-label"
            >
              Checkbox Test 2
            </label>
          </div>
        </div>
      </div>
    </div>
    <div
      className="form-group"
    >
      <div
        className="input-holder"
      >
        <div
          className="input-holder"
        >
          <div
            className="wrap-fcustominp"
          >
            <div
              className="fcustominp-state disabled"
            >
              <div
                className="fcustominp"
              >
                <input
                  disabled={true}
                  id="checkboxTest6"
                  type="checkbox"
                />
                <label
                  for="radioTest6"
                />
              </div>
              <label
                className="fcustominp-label"
                for="radioTest6"
              >
                Checkbox Test 3
              </label>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      className="form-group"
    >
      <div
        className="input-holder"
      >
        <div
          className="input-holder"
        >
          <div
            className="wrap-fcustominps-inline"
          >
            <div
              className="wrap-fcustominp"
            >
              <div
                className="fcustominp"
                onClick={[Function]}
              >
                <input
                  checked={false}
                  id="checkboxTest3"
                  type="checkbox"
                />
                <label
                  for="checkboxTest3"
                />
              </div>
              <label
                className="fcustominp-label"
              >
                Checkbox Inline Test 1
              </label>
            </div>
            <div
              className="wrap-fcustominp"
            >
              <div
                className="fcustominp"
                onClick={[Function]}
              >
                <input
                  checked={true}
                  id="checkboxTest4"
                  type="checkbox"
                />
                <label
                  for="checkboxTest4"
                />
              </div>
              <label
                className="fcustominp-label"
              >
                Checkbox Inline Test 2
              </label>
            </div>
            <div
              className="wrap-fcustominp"
            >
              <div
                className="fcustominp"
                onClick={[Function]}
              >
                <input
                  checked={false}
                  id="checkboxTest5"
                  type="checkbox"
                />
                <label
                  for="checkboxTest5"
                />
              </div>
              <label
                className="fcustominp-label"
              >
                Checkbox Inline Test 3
              </label>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Component <Checkboxes /> should renders Checkboxes with props correctly 7`] = `
<div
  className="ui-section"
  id="checkboxes"
>
  <strong
    className="ui-title"
  >
    Checkboxes
  </strong>
  <div
    className="form-group-wrapper"
  >
    <div
      className="form-group"
    >
      <div
        className="input-holder"
      >
        <div
          className="input-holder"
        >
          <div
            className="wrap-fcustominp"
          >
            <div
              className="fcustominp"
              onClick={[Function]}
            >
              <input
                checked={false}
                id="checkboxTest1"
                type="checkbox"
              />
              <label
                for="checkboxTest1"
              />
            </div>
            <label
              className="fcustominp-label"
            >
              Checkbox Test 1
            </label>
          </div>
        </div>
      </div>
    </div>
    <div
      className="form-group"
    >
      <div
        className="input-holder"
      >
        <div
          className="input-holder"
        >
          <div
            className="wrap-fcustominp"
          >
            <div
              className="fcustominp"
              onClick={[Function]}
            >
              <input
                checked={false}
                id="checkboxTest2"
                type="checkbox"
              />
              <label
                for="checkboxTest2"
              />
            </div>
            <label
              className="fcustominp-label"
            >
              Checkbox Test 2
            </label>
          </div>
        </div>
      </div>
    </div>
    <div
      className="form-group"
    >
      <div
        className="input-holder"
      >
        <div
          className="input-holder"
        >
          <div
            className="wrap-fcustominp"
          >
            <div
              className="fcustominp-state disabled"
            >
              <div
                className="fcustominp"
              >
                <input
                  disabled={true}
                  id="checkboxTest6"
                  type="checkbox"
                />
                <label
                  for="radioTest6"
                />
              </div>
              <label
                className="fcustominp-label"
                for="radioTest6"
              >
                Checkbox Test 3
              </label>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      className="form-group"
    >
      <div
        className="input-holder"
      >
        <div
          className="input-holder"
        >
          <div
            className="wrap-fcustominps-inline"
          >
            <div
              className="wrap-fcustominp"
            >
              <div
                className="fcustominp"
                onClick={[Function]}
              >
                <input
                  checked={false}
                  id="checkboxTest3"
                  type="checkbox"
                />
                <label
                  for="checkboxTest3"
                />
              </div>
              <label
                className="fcustominp-label"
              >
                Checkbox Inline Test 1
              </label>
            </div>
            <div
              className="wrap-fcustominp"
            >
              <div
                className="fcustominp"
                onClick={[Function]}
              >
                <input
                  checked={false}
                  id="checkboxTest4"
                  type="checkbox"
                />
                <label
                  for="checkboxTest4"
                />
              </div>
              <label
                className="fcustominp-label"
              >
                Checkbox Inline Test 2
              </label>
            </div>
            <div
              className="wrap-fcustominp"
            >
              <div
                className="fcustominp"
                onClick={[Function]}
              >
                <input
                  checked={true}
                  id="checkboxTest5"
                  type="checkbox"
                />
                <label
                  for="checkboxTest5"
                />
              </div>
              <label
                className="fcustominp-label"
              >
                Checkbox Inline Test 3
              </label>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Component <Checkboxes /> should renders Checkboxes with props correctly 8`] = `
<div
  className="ui-section"
  id="checkboxes"
>
  <strong
    className="ui-title"
  >
    Checkboxes
  </strong>
  <div
    className="form-group-wrapper"
  >
    <div
      className="form-group"
    >
      <div
        className="input-holder"
      >
        <div
          className="input-holder"
        >
          <div
            className="wrap-fcustominp"
          >
            <div
              className="fcustominp"
              onClick={[Function]}
            >
              <input
                checked={false}
                id="checkboxTest1"
                type="checkbox"
              />
              <label
                for="checkboxTest1"
              />
            </div>
            <label
              className="fcustominp-label"
            >
              Checkbox Test 1
            </label>
          </div>
        </div>
      </div>
    </div>
    <div
      className="form-group"
    >
      <div
        className="input-holder"
      >
        <div
          className="input-holder"
        >
          <div
            className="wrap-fcustominp"
          >
            <div
              className="fcustominp"
              onClick={[Function]}
            >
              <input
                checked={false}
                id="checkboxTest2"
                type="checkbox"
              />
              <label
                for="checkboxTest2"
              />
            </div>
            <label
              className="fcustominp-label"
            >
              Checkbox Test 2
            </label>
          </div>
        </div>
      </div>
    </div>
    <div
      className="form-group"
    >
      <div
        className="input-holder"
      >
        <div
          className="input-holder"
        >
          <div
            className="wrap-fcustominp"
          >
            <div
              className="fcustominp-state disabled"
            >
              <div
                className="fcustominp"
              >
                <input
                  disabled={true}
                  id="checkboxTest6"
                  type="checkbox"
                />
                <label
                  for="radioTest6"
                />
              </div>
              <label
                className="fcustominp-label"
                for="radioTest6"
              >
                Checkbox Test 3
              </label>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      className="form-group"
    >
      <div
        className="input-holder"
      >
        <div
          className="input-holder"
        >
          <div
            className="wrap-fcustominps-inline"
          >
            <div
              className="wrap-fcustominp"
            >
              <div
                className="fcustominp"
                onClick={[Function]}
              >
                <input
                  checked={false}
                  id="checkboxTest3"
                  type="checkbox"
                />
                <label
                  for="checkboxTest3"
                />
              </div>
              <label
                className="fcustominp-label"
              >
                Checkbox Inline Test 1
              </label>
            </div>
            <div
              className="wrap-fcustominp"
            >
              <div
                className="fcustominp"
                onClick={[Function]}
              >
                <input
                  checked={false}
                  id="checkboxTest4"
                  type="checkbox"
                />
                <label
                  for="checkboxTest4"
                />
              </div>
              <label
                className="fcustominp-label"
              >
                Checkbox Inline Test 2
              </label>
            </div>
            <div
              className="wrap-fcustominp"
            >
              <div
                className="fcustominp"
                onClick={[Function]}
              >
                <input
                  checked={false}
                  id="checkboxTest5"
                  type="checkbox"
                />
                <label
                  for="checkboxTest5"
                />
              </div>
              <label
                className="fcustominp-label"
              >
                Checkbox Inline Test 3
              </label>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
